src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +75,183 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+
+  it('should simulate update of new mappings row data', () => {
```

Comment:
Is the page being tested supposed to simulate update of nee mappings row data as part of its functionality?

Reply:
I have updated it, please review.